portfolio/src/main/java/com/google/sps/servlets/ContactMeListsServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.data.ContactMe;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/list-contactMe")
+public class ContactMeListsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("contactMe").setOrderBy(OrderBy.desc("timestamp")).build();
+    QueryResults<Entity> results = datastore.run(query);
+
+    List<ContactMe> contactMes = new ArrayList<>();
+    while (results.hasNext()) {
+      Entity entity = results.next();
+
+      long id = entity.getKey().getId();
+      String firstName = entity.getString("firstName");
+      String lastName = entity.getString("lastName");
+      String subject = entity.getString("subject");
+      String title = entity.getString("title");
+      long timestamp = entity.getLong("timestamp");
+
+      ContactMe contactMe = new ContactMe(id, title, timestamp, firstName, lastName, subject);
+      contactMes.add(contactMe);
```

Comment:
nit: line 41 and 42 can be consolidated to contactMes.add(new ContactMe(id, title, timestamp, firstName, lastName, subject));